runner/Dockerfile.ubuntu.1804

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:18.04
 
 ARG TARGETPLATFORM
-ARG RUNNER_VERSION=2.280.3
```

Comment:
This version was a few behind, was there a reason for that?

Reply:
It's just the default argument value I think. The actual value that is used for deploying is controlled by the GitHub workflow in https://github.com/actions-runner-controller/actions-runner-controller/blob/master/.github/workflows/runners.yml#L18

Reply:
It's as @reiniertimmer says, the workflow passes in the runner version to be used. Renovate bumps this every new actions/runner release. Feel free to update the versions in the Dockerfiles as we might as well but they aren't the version that gets released.

`runner/Dockerfile.ubuntu.1804` itself is a image we no longer build, it's just hasn't been removed so it's there as a point of reference.